eslint.config.mjs

```diff
@@ -0,0 +1,59 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+import { includeIgnoreFile } from "@eslint/compat";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+const gitignorePath = path.resolve(__dirname, ".gitignore");
+
+export default [
+    includeIgnoreFile(gitignorePath),
+    ...compat.extends("brightspace/lit-config").map(c => ({
+		...c,
+		files:['**/*.js','**/*.html']
+	})),
+    {
+        languageOptions: {
+            globals: {
+                Prism: false,
+            },
+        },
+    },
+    {
+        files: ['**/*.js','**/*.html'],
+        rules: {
+            "no-restricted-syntax": [0, "CatchClause[param=null]"]
+        },
+    },
+    {
+        files: ["**/demo/**.html"],
+        rules: {
+            "no-console": 0,
+        },
+    },
+    {
+        files: ["mixins/demo/**.html"],
```

Comment:
Hmm, why are the mixin demos special?

Reply:
This comes from the existing config, but it is meant for a directory that no longer exists. I looked at this [PR](https://github.com/BrightspaceUI/core/pull/3211) and it looks like the demos were created, the exceptions were added and then the demos were moved.

Reply:
Yeah whatever they were needed for clearly aren't anymore.